scripts/version-changelog.js

```diff
@@ -1,105 +0,0 @@
-#!/usr/bin/env node
```

Comment:
This file was later published as the package [`version-changelog`](https://github.com/jesstelford/version-changelog), so is no longer needed here 👍